test/e2e_federation/authn.go

```diff
@@ -44,7 +45,7 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 			framework.ExpectNoError(err)
 
 			nsName := f.FederationNamespace.Name
-			svc := createServiceOrFail(fcs, nsName, FederatedServiceName)
+			svc := createServiceOrFail(fcs, nsName, FederatedServiceName, v1.ServiceTypeClusterIP)
```

Comment:
(No action required) It there are only 2 types of services that are in use, it might be cleaner to have 2 separate methods (that both call an underlying method with the appropriate argument)>

```golang
createServiceOrFail(...)
createLBServiceOrFail(...)

```